test/env.hh

```diff
@@ -1,139 +0,0 @@
-/*
- * Copyright (C) 2021 Open Source Robotics Foundation
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *
- */
-
-// Note: this is a direct copy from ignition::common,
-// and can be removed when sdformat has common as a dependency
-
-#ifndef SDF_TEST_ENV_HH
-#define SDF_TEST_ENV_HH
-
-#include <cstdlib>
-#include <string>
-
-#ifdef _WIN32
-#include <windows.h>
-#endif
-
-namespace sdf
-{
-  namespace testing
-  {
-
-    /// \brief Find the environment variable '_name' and return its value.
-    ///
-    /// Note: the intention is to put this in ign-util, and remove it
-    /// from sdformat once the depcnency is in place
-    ///
-    /// \param[in] _name Name of the environment variable.
-    /// \param[out] _value Value if the variable was found.
-    /// \param[in] _allowEmpty Allow set-but-empty variables.
-    ///           (Unsupported on Windows)
-    /// \return True if the variable was found or false otherwise.
-    bool env(const std::string &_name,
-             std::string &_value,
-             bool _allowEmpty)
-    {
-      std::string v;
-      bool valid = false;
-#ifdef _WIN32
-      // Unused on Windows, suppress warning
-      (void) _allowEmpty;
-      const DWORD buffSize = 32767;
-      static char buffer[buffSize];
-      if (GetEnvironmentVariable(_name.c_str(), buffer, buffSize))
```

Comment:
The implementation of `env` here is different from `ign-utils`. This might be what's causing the test failures.